src/charts/css/charts.css

```
@@ -175,14 +190,15 @@ select, input, textarea {
 .ciview-chart-titlebar{
     display:flex;
     padding:4px;
-    background: var(--title_bar_color);
     color: var(--text_color);
+    background-color: var(--background_color);
     flex:0 0 auto;
     font-size:14px;
+    border: 1px solid #dfdfdf; /* Add a subtle border */
 }
```

```suggestion
    border: 1px solid rgba(120,120,120, 0.2); /* Add a subtle border */
```
this looks subtle to me on both light & dark